### DISCUSSION TITLE
Unsure of where to connect V1 & V2 in outdoor unit, cant find it?

### DISCUSSION THREAD

**Discussion by ouchkilljoys** [omerfaruk-aran/esphome_samsung_hvac_bus#126] [Questions]
I just wanted to check before I plug it into my M5STACK ATOM & RS-485 Base that these were the correct spots or not?

Is it the 1(L) 2(N) and then F1 and F2 I need to connect?

Also my unit is AC140TNHPKG if that makes a difference at all, thanks for the help!

![img](https://github.com/lanwin/esphome_samsung_ac/assets/25874008/e031a2d6-b04e-4c4b-ae56-949a561bc210)

**Comment by lanwin** [comment]
You only need F1/F2.  

Be carefull! L and N are the power wires and transport 220/110 volts. If you need power for your M5STACK you have to add an 5V converter.

You also dont need to connect the device to the outside unit. It also can be placed at one of the indoor units as long its connected to F1/F2.

**Reply by ouchkilljoys** [reply]
Oh is there a chance the F1 + F2 would also provide power to the unit as well?

And I cant use the inside wires because they are F3 + F4 :(

**Reply by ouchkilljoys** [reply]
If anyone runs into the same issue I did, I grabbed one of these https://www.altronics.com.au/p/m8420-30w-12v-2.5a-mini-screw-terminal-power-supply/ and then hooked it up to 1(L) and 2(N) to the converter and then -Vo and +Vo output to the V1 and V2 ports respectively on the RS-485 and then the F1 and F2 in their usual spots.

**Reply by lanwin** [reply]
30W should be a lot more then you need.
